docs/development_suite/api-console/api-design/listeners.md

```diff
@@ -6,16 +6,71 @@ sidebar_label: Create Listeners
 
 ## What is a Listener
 
-A listener is an **entry point** for external connections directed to your cluster. Listeners are configured on your API Gateway, where they are mapped to a single port, and define rules to route incoming requests from external clients to a resource in your cluster, such as a specific microservice, exposed through endpoints.
+A Listener is a logical entity configured on your API Gateway that checks for incoming connection requests directed to your cluster.  
+Each listener is mapped to a single port and defines rules to route incoming requests from external clients to a resource in your cluster, such as a specific microservice, exposed through endpoints.
 
-The purpose of this page is to describe all the functionalities that you can find in the **Listeners** section of the [Design area](/development_suite/api-console/api-design/overview.md) of the Mia-Platform Console.
+This page describes the features of a Listener and how to configure these from the **Listeners** section of the [Design area](/development_suite/api-console/api-design/overview.md) of the Console.
+
+:::info
+Currently Listeners are available only to Projects configured to use an **Envoy API Gateway**.
+:::
 
 ## Create a Listener
 
-:::info
-The listeners feature is only supported in combination with an **Envoy API Gateway**, which needs to be instantiated on your Project.
+Follow this three steps in order to create the new Listener and make an Endpoint reachable from outside the Console:
+- create the new Listener resource
+- expose the Endpoint on the Listener
+- define a new Traefik IngressRoute for the newly exposed API Gateway port
```

Comment:
this depends on the ingress route you are using; I'd put this into a warning box that remembers the user that it may need to configure their cluster ingress to route traffic towards the newly defined listener port, e.g. using traefik you may want to add a new route in the IngressRoute resource.

Reply:
made both here and description below more generic on Ingress configuration and added an example for traefik